src/main/kotlin/no/nav/syfo/domain/PersonIdentNumber.kt

```diff
@@ -0,0 +1,11 @@
+package no.nav.syfo.domain
+
+data class PersonIdentNumber(val value: String) {
```

Comment:
Blir det uten `Number` her, eller landet du på å beholde det?
Og skulle vi vurdert å legge til ordentlig validering når vi først har en klasse, eller kanskje det er unødvendig? 🤔

Reply:
1. Ikke landet noe på navn ennå.
2. Når du nevner "ordentlig validering", mener du full validering av hele nummeret, fødelsdato-sifre, kjønsssifer, kontroll-siffer?

Reply:
Jeg tenkte med kontrollsiffer o.l. ja 👍 Hva mener du med "egen greie"? Jeg tenkte at hvis vi gidder å legge det til, så passer det fint her, mtp. at vi allerede "validerer" identen i dag 🤔

Reply:
Med egen greie, tenkte jeg på lage det i en egen PR/feature. Tar lit tid å lage det, er mye greier som trengs. Er også mulig at noen andre har laget en pakke på dette, men da sannsynligvis ikke med Kotlin data-class.

Reglene for F-nummer og D-nummer kan man se her:
https://no.wikipedia.org/wiki/F%C3%B8dselsnummer

I hvilken grad disse reglene da blir favne om NPID-nummer er jo usikker på.